generator/internal/scaffold_generator.cc

```diff
@@ -138,10 +138,10 @@ std::map<std::string, std::string> ScaffoldVars(
   vars["library"] = library;
   vars["product_namespace"] = absl::StrReplaceAll(
       absl::StripSuffix(
-          absl::StrCat(library, "_",
+          absl::StrCat(library, "/",
```

Comment:
why did this change?

Reply:
nvm

Reply:
It's clearer to think of `LibraryName()` and `ServiceSubdirectory()` as paths, so join them with a slash.